packages/browser/src/Widget/Server/SPARQL/Service/SparqlEndpointService.js

```diff
@@ -34,9 +41,9 @@ export class SparqlEndpointService extends RequestService {
    */
   async querySparqlEndpoint(query) {
     const full_url = this.url + this.url_parameters + encodeURIComponent(query);
-    const options = {};
 
-    const request = await this.request('GET', full_url, options);
+    // TODO: add mechanism for configuring POST requests
```

Comment:
I think we should not use the request service (which is a util class used in a smdb context) but directly the web api [XMLHttpRequest](https://developer.mozilla.org/fr/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest) or [fetch](https://developer.mozilla.org/fr/docs/Web/API/Fetch_API/Using_Fetch)

Reply:
fetch was used and this class was heavily simplified and integrated into the SparqlEndpointResponseProvider